css_tasks/task11/js/slider.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable max-len */
+
+// Probably it could be simplified
+
+import {elements} from './elements.js';
+import {constants} from './constants.js';
+export function slider() {
+  elements.then(
+      ({
+        firstCarousel,
+        secondCarousel,
+        firstCarouselItems,
+        secondCarouselItems,
+      }) => {
+        let {slideIndex} = constants;
```

Comment:
you change this value, it is not constant, just variable with some default value

Reply:
Moreover, it's declaration of new variable with the same name. The initial `slideIndex` in the file is unchanged.